templates/books-index.html

```diff
@@ -0,0 +1,9 @@
+<div ng-repeat="livros in booksIndexCtrl.livros.books">
+
+	<p ng-hide="editing"><a ng-href="/books/{{livros._id}}">{{livros.title}}</a></p>
```

Comment:
Why does this need ng-hide? You never set editing. This seems left-over from earlier code, like you copy/pasted without understanding what it did.